data/campaigns/Under_the_Burning_Suns/units/quenoth/Youth.cfg

```diff
@@ -317,8 +317,18 @@ _"Nym is Kaleh’s childhood friend, a young, rebellious lady with quick wits an
 #enddef
 
 #define KALEH_ANIMATIONS
+    {LEADING_ANIM "units/quenoth/kaleh-lead2.png" "units/quenoth/kaleh-lead1.png" (12,-32)}
     {DEFENSE_ANIM_RANGE "units/quenoth/kaleh-defend-2.png" "units/quenoth/kaleh-defend-1.png" {SOUND_LIST:ELF_HIT} melee}
-    {DEFENSE_ANIM_RANGE "units/quenoth/kaleh-bow-defend-2.png" "units/quenoth/kaleh-bow-defend-1.png" {SOUND_LIST:ELF_HIT} ranged}
+    {DEFENSE_ANIM_FILTERED "units/quenoth/kaleh-bow-defend-2.png" "units/quenoth/kaleh-bow-defend-1.png" {SOUND_LIST:ELF_FEMALE_HIT} (
```

Comment:
Several of Kaleh's sounds have changed from ELF_HIT to ELF_FEMALE_HIT, I guess that's a copy-paste error.

(I'm playing with adding Kaleh and Nym to the tutorial campaign.)